tests/test_bioimg.py

```diff
@@ -19,9 +19,11 @@ def test_validate_io_paths_accepted(self):
                 ["s3://test_out/b", "s3://test_out/d"],
             ),
         }
-        for test_name, (source, dest) in accepted_pairs.items():
-            with self.subTest(f"case: {test_name}"):
-                validate_io_paths(source, dest)
+        register = {"true": True, "false": False}
+        for _, r in register.items():
```

Comment:
`for r in (True, False)`?

Reply:
Yeap just wanted to make this more readable for the one that reads the tests.

Reply:
Given that, in this case, the "name" is literally just "true" or "false", I'm not sure this does that; I find it just adds a layer of indirection. Maybe making the `r` value be named `register` (or `for_registration` as above) would communicate the intent better?

Reply:
No worries will change.